openassessment/locale/config.yaml

```diff
@@ -74,11 +74,19 @@ dummy_locales:
 
 # Directories we don't search for strings.
 ignore_dirs:
```

Comment:
Yeah I might see if you can watch the output on this branch and see if it's actually excluding these directories.